.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    ignore:
+      # core is using lower Guava version, don't touch it unless you want things to blow up on client installations at runtime
+      - dependency-name: "com.google.guava:guava"
```

Comment:
Why is this here? This plugin’s `pom.xml` makes no reference to Guava. I think this whole `ignore` section can be deleted.